packages/commerce-sdk-react/src/hooks/ShopperCustomers/query.ts

```diff
@@ -50,18 +49,59 @@ export const useCustomerAddress = (
     const {shopperCustomers: client} = useCommerceApi()
     return useQuery(['address', arg], () => client.getCustomerAddress(arg))
 }
+type UseCustomerBasketsParameters = NonNullable<
+    Argument<Client['getCustomerBaskets']>
+>['parameters']
+type UseCustomerBasketsHeaders = NonNullable<Argument<Client['getCustomerBaskets']>>['headers']
+type UseCustomerBasketsArg = {
+    headers?: UseCustomerBasketsHeaders
+    rawResponse?: boolean
+} & UseCustomerBasketsParameters
 /**
  * A hook for `ShopperCustomers#getCustomerBaskets`.
  * Gets the baskets of a customer.
  * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-customers?meta=getCustomerBaskets} for more information about the API endpoint.
  * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shoppercustomers.shoppercustomers-1.html#getcustomerbaskets} for more information on the parameters and returned data type.
  * @returns An object describing the state of the request.
  */
-export const useCustomerBaskets = (
-    arg: Argument<Client['getCustomerBaskets']>
-): UseQueryResult<DataType<Client['getCustomerBaskets']>, Error> => {
-    const {shopperCustomers: client} = useCommerceApi()
-    return useQuery(['baskets', arg], () => client.getCustomerBaskets(arg))
+export function useCustomerBaskets(
```

Comment:
this doesn't belong to this PR, but i had to add it to unblock my PR. Implementation is taken from https://github.com/SalesforceCommerceCloud/pwa-kit/pull/762.

I will remove this from the PR after review.